modules/recon/credentials-credentials/hashes_org.py

```diff
@@ -9,24 +9,26 @@ class Module(BaseModule):
         'name': 'Hashes.org Hash Lookup',
         'author': 'Tim Tomes (@LaNMaSteR53) and Mike Lisi (@MikeCodesThings)',
         'version': '1.0',
-        'description': 'Uses the Hashes.org API to perform a reverse hash lookup. Updates the \'credentials\' table with the positive results.',
+        'description': 'Uses the Hashes.org API to perform a reverse hash lookup. Updates the \'credentials\' table '
+                       'with the positive results.',
         'required_keys': ['hashes_api'],
         'comments': (
-            'Hash types supported: MD5, MD4, NTLM, LM, DOUBLEMD5, TRIPLEMD5, MD5SHA1, SHA1, MYSQL5, SHA1MD5, DOUBLESHA1, RIPEMD160',
-            'Hashes.org is a free service. Please consider a small donation to keep the service running. Thanks. - @s3inlc'
+            'Hash types supported: MD5, MD4, NTLM, LM, DOUBLEMD5, TRIPLEMD5, MD5SHA1, SHA1, MYSQL5, SHA1MD5, '
+            'DOUBLESHA1, RIPEMD160',
+            'Hashes.org is a free service. Please consider a small donation to keep the service running. Thanks. - '
+            '@s3inlc'
         ),
-        'query': 'SELECT DISTINCT hash FROM credentials WHERE hash IS NOT NULL AND password IS NULL AND type IS NOT \'Adobe\'',
+        'query': 'SELECT DISTINCT hash FROM credentials WHERE hash IS NOT NULL AND password IS NULL AND type IS '
+                 'NOT \'Adobe\'',
     }
 
     def module_run(self, hashes):
         api_key = self.keys.get('hashes_api')
-        url = 'https://hashes.org/api.php'
-        payload = {'key':api_key}
         for hashstr in hashes:
-            payload['query'] = hashstr
+            url = f"https://hashes.org/api.php?key={api_key}&query={hashstr}"
```

Comment:
While I know this will work, why did we move away from Python Request's query string encoding for `payload` and hard code everything in the URL? By using the `payload` the way we did before, it made it more modular with regards to URL changes, etc. Just curious.

Reply:
This was based on the hashes.org v2.0 documentation and how they had the examples. Using it the previous way was broken not sure if maybe they updated the API but when I tried using the module the previous way I was getting an invalid API Key error from a valid API Key. Once I switched it everything worked. Pushing back up removed the PEP8 line breaks that you didn't like.

Reply:
Oh, that was a PEP8 thing? Crap. I need to review that again I guess. Forgot about the line wrap thing. I'm good with PEP8. Feel free to call me out on OCD comments that go against PEP8. That's the standard, not me.